package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "nucypher-contracts",
+  "version": "0.0.1",
```

Comment:
Note that there's an existing "0.1.0" version on https://www.npmjs.com/package/nucypher-contracts, but we may want to just remove it since it would be good to have `nucypher-contracts` on NPM to be organized into the `@nucypher` npm organization.